### PR TITLE
4866-ClyFullBrowser-classworldMenuOn-checks-if-nautilus-is-in-the-image

### PR DESCRIPTION
**Pull request by myroslavarm**
#4866 removed the ifTrue check for Nautilus as it's not in the system anymore

### CODE REVIEW

**Review by jecisc — COMMENTED** [review]
There is still a check for Nautilus just above. 
`self environment hasClassNamed: #Nautilus`

**Comment by myroslavarm** [comment]
@jecisc removing that line makes calypso pop out of tools into the main "world" context menu which i'm guessing is far from ideal. i left the ifFalse checks because logically it makes sense that when it does not find nautilus (which it doesn't) that it does some other behaviour

**Comment by dionisiydk** [comment]
@myroslavarm I also think all Nautilus part needs to be removed. If there is a side effect from it then it needs to be fixed

File: src/Calypso-SystemTools-FullBrowser/ClyFullBrowser.class.st [hunk 1]
```diff
@@ -146,19 +146,7 @@ ClyFullBrowser class >> taskbarIconName [
...
-	(isDefaultBrowser and: [ self environment hasClassNamed: #Nautilus ])
-		ifFalse: [ 
-			target := self.
-			targetName := 'Calypso' ].
-	(aBuilder item: (targetName, ' browser') asSymbol)	
-		order: 0.5;
-		help: targetName, ' browser';
-		icon: (self iconNamed: #nautilus);
-		action: [ target open ]
+	^ self
```
Comment (jecisc) [inline src/Calypso-SystemTools-FullBrowser/ClyFullBrowser.class.st]: Then now the method can be totally removed :)

**Review by jecisc — CHANGES_REQUESTED** [review]

**Review by dionisiydk — COMMENTED** [review]
Ah, exactly. It was just to override default browser menu

**Pull request merged**